app/src/main/java/com/keylesspalace/tusky/fragment/ViewThreadFragment.java

```diff
@@ -532,20 +532,14 @@ private void showCard(Card card) {
         }
     }
 
-    public void clear() {
-        statuses.clear();
-        adapter.clear();
-    }
-
     private void updateRevealIcon() {
         ViewThreadActivity activity = ((ViewThreadActivity) getActivity());
         if (activity == null) return;
 
         boolean hasAnyWarnings = false;
         // Statuses are updated from the main thread so nothing should change while iterating
         for (int i = 0; i < statuses.size(); i++) {
-            if (statuses.get(i).getSpoilerText() != null
-                    && !statuses.get(i).getSpoilerText().isEmpty()) {
+            if (!TextUtils.isEmpty(statuses.get(i).getSpoilerText())) {
```

Comment:
TextUtils is a platform class which will make things hard to test. Maybe use StringsKt or something?